lib/saxerator/configuration.rb

```diff
@@ -30,23 +26,23 @@ def generate_key_for(val)
     end
 
     def hash_key_normalizer
-      @hash_key_normalizer ||= lambda { |x| x.to_s }
+      @hash_key_normalizer ||= -> (x) { x.to_s }
```

Comment:
The current build matrix includes Ruby 1.9.3 which does not have stabby lambdas.

Honestly, though, I'm ok dropping 1.9.3 support. It's a dead ruby at this point.